### PR TITLE
I finished the section Recommendations

### PR DESCRIPTION
**Pull request by Constantin-Matei**

### CODE REVIEW

**Review by biggicode — APPROVED** [review]

**Review by vasilemaftita — APPROVED** [review]
Nice; Work...

File: src/_component/recommendations/recommendations.pug [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#recomamendations
+- var sectionTitle = data.recommendations.sectionTitle
```
Comment (QuintyHH) [inline src/_component/recommendations/recommendations.pug:2]: Am I insane or does this not get used anywhere?
Also, love the Id :D #Recomamendations

Comment (QuintyHH) [reply src/_component/recommendations/recommendations.pug:2]: Ah I see what is happening, it gets used by the following inclusion of `../sectiunea-titlu/sectiunea-titlu`

**Review by QuintyHH — APPROVED** [review]
💯

**Pull request merged**